tools/perf/benchmark.common.js

```diff
@@ -1,19 +1,8 @@
-
-function runTests(env) {
-  const testData = JSON.parse(env.readFile("./fixtures/benchmarks.json"));
-
+function runTest(env) {
   const results = {};
-
-  for (let testName in testData) {
-    let test = testData[testName];
-    runTest(env, testName, test.args, test.functions, results);
-  }
-
-  return results;
-}
-
-function runTest(env, name, args, fncs, results) {
-  const ftlCode = env.readFile(`./fixtures/${name}.ftl`);
+  const testData = JSON.parse(env.readFile("./fixtures/benchmarks.json"));
```

Comment:
Maybe an idea for a future improvement, not blocking this PR: could we read this file once and pass it to `runTest`?

Reply:
I would prefer not to pass complex data such as args/callbacks via command line tbh.